.drone.star

```diff
@@ -1953,6 +1950,7 @@ def setupServerAndApp(ctx, phpVersion, logLevel, federatedServerNeeded = False,
             "php occ a:l",
             "php occ a:e %s" % ctx.repo.name if enableApp else "",
             "php occ a:e testing",
+            "php occ a:disable notifications",
```

Comment:
We only want to do this where we have to - just for PHP unit tests.
I suggest put this in the top config "phpunit" section for each unit test pipeline.
```
            "extraCommandsBeforeTestRun": [
                "php occ a:disable notifications",
            ],

```

And also that means that we do not have to touch the common starlark code. We can do this just by extra in the config section.